src/login/logind-button.c

```diff
@@ -495,6 +640,8 @@ int button_open(Button *b) {
 
         (void) button_set_mask(b->name, fd);
 
+        button_get_modifier_mask(b);
```

Comment:
i think we should fail button_open() if this call here fails, there's no reason why it could rightly fail